test/migrator_test.rb

```diff
@@ -29,48 +28,48 @@
     end
 
     # now test down/ up
-    ActiveRecord::Migrator.run(:down, migration_path, 20110824010216)
+    migrator(:down, 'migrations', 20110824010216).run
     ActiveRecord::Base.on_all_shards do
       assert !table_has_column?("tickets", "sharded_column")
     end
 
-    ActiveRecord::Migrator.run(:down, migration_path, 20110829215912)
+    migrator(:down, 'migrations', 20110829215912).run
     ActiveRecord::Base.on_shard(nil) do
       assert !table_has_column?("accounts", "non_sharded_column")
     end
 
-    ActiveRecord::Migrator.run(:up, migration_path, 20110824010216)
+    migrator(:up, 'migrations', 20110824010216).run
     ActiveRecord::Base.on_all_shards do
       assert table_has_column?("tickets", "sharded_column")
     end
 
-    ActiveRecord::Migrator.run(:up, migration_path, 20110829215912)
+    migrator(:up, 'migrations', 20110829215912).run
     ActiveRecord::Base.on_shard(nil) do
       assert table_has_column?("accounts", "non_sharded_column")
     end
   end
 
   it "does not migrate bad migrations" do
-    migration_path = File.join(File.dirname(__FILE__), "/cowardly_migration")
     assert_raises StandardError do
-      ActiveRecord::Migrator.migrate(migration_path)
+      migrator(:up, 'cowardly_migration').migrate
     end
   end
 
   it "fails with failing migrations" do
     # like, if you have to break a migration in the middle somewhere.
-    migration_path = File.join(File.dirname(__FILE__), "/failure_migration")
-
-    assert failure_migration_pending?(migration_path)
+    assert failure_migration_pending?('failure_migration')
     begin
-      ActiveRecord::Migrator.migrate(migration_path)
-    rescue
+      migrator(:up, 'failure_migration').migrate
+    rescue => e
+      unless e.message.include?("FAIL")
```

Comment:
might be nice to make this more specific and add a comment on what exactly it's catching